tests/contrib/open_telemtry/test_open_telemetry_middleware.py

```diff
@@ -0,0 +1,129 @@
+from time import sleep
+from typing import Any, Tuple, cast
+
+from opentelemetry.metrics import get_meter_provider, set_meter_provider
+from opentelemetry.sdk.metrics._internal import MeterProvider
+from opentelemetry.sdk.metrics._internal.aggregation import (
+    ExplicitBucketHistogramAggregation,
+)
+from opentelemetry.sdk.metrics._internal.export import InMemoryMetricReader
+from opentelemetry.sdk.metrics._internal.instrument import Counter
+from opentelemetry.sdk.resources import SERVICE_NAME, Resource
+from opentelemetry.sdk.trace import Span, TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor
```

Comment:
You can use `SimpleSpanProcessor` for testing.

Reply:
ok, I didnt see this one. Ill check it out.